code_examples/ApacheKafka/prototype/kafka_services/README.md

```diff
@@ -0,0 +1,75 @@
+# Kafka Services
```

Comment:
I am missing information on what the two services provided here are actually doing. Besides that, I don't feel that providing binary web services which the user should just blindly run is a good idea.